### PR TITLE
added try catch around Microsoft translator

### PR DESCRIPTION
**Pull request by jsparvath**
added try catch around Microsoft translator

### CODE REVIEW

File: python_translators/translators/best_effort_translator.py [hunk 1]
```diff
@@ -34,19 +34,25 @@ def __init__(self, source_language: str, target_language: str):
...
 
             # Microsoft Translator WITH context
-            t = MicrosoftTranslatorFactory.build_with_context(**lang_config)
-            t.quality = 80
-            self.add_translator(t)
+            try:
+                t = MicrosoftTranslatorFactory.build_with_context(**lang_config)
+                t.quality = 80
+                self.add_translator(t)
+            except:
```
Comment (mircealungu) [inline python_translators/translators/best_effort_translator.py]: Can't you use a more specific except clause?

File: python_translators/translators/google_translator.py [hunk 2]
```diff
@@ -35,7 +36,11 @@ def __init__(self, source_language: str, target_language: str, key: str, transla
         )
 
         self.key = key
-        self.translation_service = build('translate', 'v2', developerKey=key)
+        try:
+            self.translation_service = build('translate', 'v2', developerKey=key)
+        except HttpError as e:
+            responseString = f'error: {e} rrrrrrrrrrrresponse: {e.resp}  ssssssssssstatus: {e.resp.status}'
```
Comment (mircealungu) [inline python_translators/translators/google_translator.py]: is this a pirate log? :)

Comment (jsparvath) [reply python_translators/translators/google_translator.py]: hahah :smile: 
A no one was supposed to see that..

**Comment by jsparvath** [comment]
Closing pull request and making an issue instead, as I have some concerns I want to clarify before implementing a solution.

**Pull request closed without merging**